packages/rocketchat-slashcommands-tokenpass/package.js

```diff
@@ -0,0 +1,34 @@
+Package.describe({
```

Comment:
There is no need for another package just for adding slash commands. I would recommend moving this package's files to the existing tokenpass package.

Reply:
I was believe that specific packages for slash commands was a good practice on RC architecture :-) So I'll move this.